examples/08-python-operators/02-python_operators_with_dependencies.py

```diff
@@ -58,11 +58,12 @@
 # created for you.
 
 import os
+from pathlib import Path
 
 from ansys.dpf.core import examples
 
-plugin_path = examples.download_gltf_plugin()
-folder_root = os.path.join(os.getcwd().rsplit("pydpf-core", 1)[0], "pydpf-core")
+plugin_path = Path(examples.download_gltf_plugin())
+folder_root = Path(__file__).parent.parent.parent
```

Comment:
This line is the one making the CI/CD to fail. I think the problem is that, this is converted to a notebook by Sphinx gallery and thus, the variable `__file__` is no longer present, @moe-ad